data-access/src/app/application-services/users/end-user/end-user.domain.ts

```diff
@@ -28,9 +28,9 @@ export class EndUserDomainApiImpl
       if (!domainObject || domainObject.externalId !== this.context.verifiedUser.verifiedJWT.sub) {
         throw new Error('Unauthorized');
       }
-      if (user.personalInformation?.identityDetails?.restOfName !== undefined) domainObject.personalInformation.identityDetails.restOfName=(user.personalInformation?.identityDetails?.restOfName);
-      if (user.personalInformation?.identityDetails?.lastName !== undefined) domainObject.personalInformation.identityDetails.lastName=(user.personalInformation?.identityDetails?.lastName);
-      if (user.personalInformation?.contactInformation?.email !== undefined) domainObject.personalInformation.contactInformation.email=(user.personalInformation?.contactInformation?.email);
+      if (user.personalInformation?.identityDetails?.restOfName !== undefined) domainObject.personalInformation.identityDetails.RestOfName=(user.personalInformation?.identityDetails?.restOfName);
```

Comment:
For one liner condtionals in the domain datasources, we prefer to keep them on one line to avoid the file becoming too big for some of the larger update mutations

Reply:
No need for braces and no need for comments. It's our convention and we are sticking to it. Please disregard this use braces code quality in the future